src/scripts/main.js

```diff
@@ -1,3 +1,46 @@
 'use strict';
 
-// write code here
+const liElements = document.querySelectorAll('li');
+const ulElement = document.querySelector('ul');
+const arr = [...liElements];
+
+function toNumb(salary) {
+  return +salary.slice(1).replace(',', '');
+}
+
+const sortList = function(list) {
```

Comment:
Why do you use two different ways of creating a function here and above? It can be confusing for other developers